duplicacy-util.go

```diff
@@ -413,17 +413,17 @@ func performBackup() error {
 
 	// Perform backup/copy operations if requested
 	if cmdBackup {
-		for i := range configFile.backupInfo {
+		for _, backupInfo := range configFile.backupInfo {
```

Comment:
I like what you've done here. Two things to be certain of:

1. Will order of items in the range be identical if the range has multiple items (not certain, I'd need to check to be sure, but it's probably the same), and

2. In the same section, for `copy`, `prune`, and `check` operations, the old construct was used. Can you change those constructs as well?

I'm also picky about commits. I'm fine with two commits to contain all this, but review comments should be incorporated into existing commits.

Reply:
The range should be iterating the same as before, I'm just capturing the value and discarding the index, `backupInfo` should be equivalent to the old `configFile.backupInfo[i]` in this case. I can update the other `for` loops to match